<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <layout class="QGridLayout" name="mainGridLayout" rowstretch="0,0,1,0,0,0,0,0,0">
      <item row="7" column="3">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>SA</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Board size</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBox">
        <property name="minimum">
         <number>4</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="QLineEdit" name="lineEdit_5"/>
      </item>
      <item row="7" column="4">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>GA</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>BT</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="lineEdit_3"/>
      </item>
      <item row="7" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>TS</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QLineEdit" name="lineEdit_4"/>
      </item>
      <item row="8" column="4">
       <widget class="QLineEdit" name="lineEdit_6"/>
      </item>
      <item row="0" column="5" rowspan="7">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Algorithm</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QRadioButton" name="btRadioButton">
             <property name="text">
              <string>BT</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="tsRadioButton">
             <property name="text">
              <string>TS</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="saRadioButton">
             <property name="text">
              <string>SA</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="gaRadioButton">
             <property name="text">
              <string>GA</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Queens</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="solveButton">
        <property name="text">
         <string>Solve</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="queensLineEdit"/>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="nextStepButton">
        <property name="text">
         <string>Next Step</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>585</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
